src/components/KCollapse/KCollapse.vue

```diff
@@ -37,7 +35,7 @@
                 data-testid="k-collapse-trigger-label"
               >
                 <KIcon
-                  class="k-collapse-trigger-chevron mr-1"
+                  class="k-collapse-trigger-chevron"
                   :icon="collapseIcon"
                   size="14"
```

Comment:
```suggestion
                  :size="KUI_ICON_SIZE_20"
```
issue: icon size needs to be replaced with a token (you'll need to import the const though)

Reply:
Should I also add in a comment about the size? Since, there is no icon size 14 and hence will be using size 12 here.

Reply:
Not necessary. Just check visually to make sure the new icon size doesn't break anything or look very off. If it's all good it's expected change